explorer/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+authors = ["dev@iohk.io"]
+description = "explorer service for jormungandr"
+documentation = "https://github.com/input-output-hk/jormungandr#USAGE.md"
+edition = "2018"
+homepage = "https://github.com/input-output-hk/jormungandr#README.md"
+license = "MIT OR Apache-2.0"
+name = "explorer"
+repository = "https://github.com/input-output-hk/jormungandr"
+version = "0.9.1"
+
+[dependencies]
+futures = "0.3.5"
+futures-channel = "0.3.5"
+futures-util = "0.3.5"
+async-graphql = "2.5.1"
+async-graphql-warp = "2.6.0"
+serde = {version = "1.0.114", features = ["derive"]}
+serde_json = "1.0.56"
+serde_yaml = "0.8.13"
+structopt = "0.3.15"
+thiserror = "1.0.20"
+anyhow = "1.0.41"
```

Comment:
Both are used, since I found it quite convenient for the top level (user) errors, in which I only wanted an error message for the user, but it's not strictly necessary.

like, with these things:

```rust
        let mut client = Client::connect(settings.node.clone())
            .await
            .context("Couldn't establish connection with node")
            .map_err(Error::UnrecoverableError)?;

        let sync_stream = client
            .sync_multiverse(vec![])
            .await
            .context("Failed to establish bootstrap stream")
            .map_err(Error::UnrecoverableError)?;

        let block_events = client
            .block_subscription()
            .await
            .context("Failed to establish block subscription")
            .map_err(Error::UnrecoverableError)?;

        let tip_events = client
            .tip_subscription()
            .await
            .context("Failed to establish tip subscription")
            .map_err(Error::UnrecoverableError)?;
```

Since they are in the `main` function, having enum variants for each case is not really that important (because no one will pattern match on them), and IMO being able to just provide some specific info directly in there makes it easier to provide good error messages since otherwise errors tend to get more generic.

That said, it's certainly possible to achieve the same with just `thiserror`, and it's not used extensively, but I don't think they are really alternatives, I feel it's more like a distinction between library errors vs application errors.